test/controllers/activities_controller_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class ActivitiesControllerTest < ActionController::TestCase
+  test "should get create" do
+    get :create
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get :edit
+    assert_response :success
+  end
+
+  test "should get index" do
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.